Tema-1/script3.js

```diff
@@ -0,0 +1,34 @@
+console.log("EXERCITIUL 3")
+
+let Luni;
```

Comment:
aici poti sa declari si sa atribui in aceeasi variabila
```let Luni = 0;```

Reply:
Am facut si asa, dar am modificat atat la ex 3 cat si 4, ca ceva nu-mi dadea bine si ma gandeam ca poate n-am formulat cum trebuie